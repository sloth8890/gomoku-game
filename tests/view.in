view
place D1
view
place A19
view
place S2
view
resign
