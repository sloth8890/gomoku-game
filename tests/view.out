J10,.................................................
B3,xx.....xx.....xx.....xx.....xx.....xx...#.xxxxxxx
M16,.................................................
E17,xxxxxxx..........................................
Black wins!
D1A19S2
Thank you for playing!
